CHANGELOG.md

```diff
@@ -7,8 +7,11 @@ Scalyr Agent 2 Changes By Release
 Packaged by Tomaz Muraus <tomaz@scalyr.com> on Dec 2, 2020 14:00 -0800
 --->
 
+Bug fixes:
+* Fix to make sure we don't expect valid Docker socket when running the kubernetes monitor in CRI mode.
```

Comment:
We should always describe bugs in terms of the symptoms that customers would think in.  So, let's change this to:

"Fix to make sure we don't expect a valid Docker socket when running Kubernetes monitor in CRI mode.  This fixes an issue preventing the K8s monitor from automatically failing over to CRI mode if Docker is not available."

Reply:
I dont think thats actually accurate though, because we dont fail over to CRI and never did. I guess saying "preventing the K8s monitor from running in CRI mode if Docker is not available." would work?

Reply:
Hmm, ok, let's make sure we are on the same page on this one.  We should be auto-detecting whether or not we should run using docker or CRI.  The requirement from the customer is they want one configuration that works in both environments.  We have these options `k8s_always_use_docker` and `k8s_always_use_cri` that both default to `False`.  From my understanding, if both of those are false and docker is not available, then we should automatically use CRI.  So, one configuration would work in both environments.

Am I wrong in my understanding?

Reply:
We are auto-detecting yes, I just think failover is the wrong term for it, since we don't default to Docker, we just pick the correct one from the start automatically. It's a minor detail now that I'm typing it out.

Reply:
Ok, no worries, it is better to double check these things.  Go ahead and make the documentation change as you suggested and submit.  Thanks for doing this.